charts/victoria-metrics-alert/templates/alertmanager-deployment.yaml

```diff
@@ -72,6 +72,12 @@ spec:
           ports:
             - name: web
               containerPort: 9093
+          {{- if .Values.alertmanager.envFrom }}
```

Comment:
Please also add this new `envFrom` field in [values.yaml](https://github.com/VictoriaMetrics/helm-charts/blob/d33d44167f236e2e5f04ae4630283e53e6183600/charts/victoria-metrics-alert/values.yaml#L315).